.editorconfig

```diff
@@ -35,3 +35,192 @@ trim_trailing_whitespace = false
 indent_style = space
 indent_size = 2
 trim_trailing_whitespace = false
+
+# C# files
+[*.cs]
+
+#### Core EditorConfig Options ####
+
+# New line preferences
+insert_final_newline = true
+
+#### .NET Coding Conventions ####
+
+# Organize usings
+dotnet_separate_import_directive_groups = false
+dotnet_sort_system_directives_first = true
```

Comment:
📝 This line is the one that triggered this pull request. My IDE settings have this set to `false`, so I was unable to use the Remove and Sort Usings feature without incorrectly reordering them.